marketplace/connect/client.py

```diff
@@ -0,0 +1,77 @@
+import json
+import requests
+
+from django.conf import settings
+
+
+class ConnectAuth:
+    def get_auth_token(self) -> str:
+        request = requests.post(
+            url=settings.OIDC_OP_TOKEN_ENDPOINT,
+            data={
+                "client_id": settings.OIDC_RP_CLIENT_ID,
+                "client_secret": settings.OIDC_RP_CLIENT_SECRET,
+                "grant_type": "client_credentials",
+            },
+        )
+        token = request.json().get("access_token")
+        return f"Bearer {token}"
+
+    def auth_header(self, headers: dict = {}) -> dict:
+        if "Authorization" not in headers:
+            headers["Authorization"] = self.get_auth_token()
+        return headers
+
+
+class ConnectProjectClient(ConnectAuth):
+
+    base_url = settings.CONNECT_ENGINE_BASE_URL
+
+    def list_channels(self, channeltype_code: str):
+        channels = []
+
+        payload = {"channel_type": channeltype_code}
+        request = requests.get(
```

Comment:
Change the name of the variable to response, since it is an instance of Response and not Request.
This applies to the following line: [49](https://github.com/weni-ai/weni-integrations-engine/pull/154/files#diff-1be15eeb5dbb3513b189266691c6b4af6e9c40df745b24f0b2a5d17e844cfeedR49)

Reply:
done!